src/main/java/mate/controller/authentication/LoginController.java

```diff
@@ -0,0 +1,42 @@
+package mate.controller.authentication;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.exception.AuthenticationException;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.AuthenticationService;
+
+public class LoginController extends HttpServlet {
+    private static final String SESSION_ATTRIBUTE = "driver_id";

```

Comment:
```suggestion
    private static final String DRIVER_ID = "driver_id";

```
That const name will be more informative